vue/src/Version/VersionEdit.vue

```diff
@@ -224,12 +224,13 @@ export default defineComponent({
       NotificationsStore.remove(notificationId);
       NotificationsStore.remove('ajaxHelper');
     },
-    showNotification(message: string, context: NotificationType['context']) {
+    showNotification(message: string, context: NotificationType['context'],
+      type: null|NotificationType['type'] = null) {
       const notificationInstanceId = NotificationsStore.show({
         message,
         context,
         id: notificationId,
-        type: 'transient',
+        type: type !== null ? type : 'toast',
```

Comment:
@snake14 Should we keep `transient` for versions ? as versions are only editable if you have right to publish

Reply:
@AltamashShaikh I thought about that, but only 1 of the 4 notifications generated by this view contain a CTA. In trying to address the original issue of notifications being displayed too long, I made every notification a toast type which didn't have a CTA.